src/components/menu/menu-item/menu-item.component.js

```diff
@@ -32,6 +33,11 @@ const MenuItem = ({
   onKeyDown,
   variant = "default",
   showDropdownArrow = true,
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Why the need for this prop if it's not being used?

Reply:
Yeah thats a good shout